qiskit/pulse/instructions/directives.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Directives are hints to the pulse compiler for how to process its input programs."""
+from abc import ABC
+from typing import Optional
+
+from qiskit.pulse import channels as chans
```

Comment:
Why?

Reply:
`channels` is defined as an argument already for instructions.

Reply:
Can you do `from qiskit.pulse.channels import Channel` instead, so that `chans` and `channels` don't get confused, please?